### ISSUE TITLE
Config escaping "<" in velocity motd

### ISSUE THREAD

**Issue by TropicalShadow** [jeremylvln/Shulker#382]
### What happened?

# Summary
The configuration process is unexpectedly adding a double backslash before the '<' character when setting values for MOTD. This behavior is not in line with the expected functionality of MiniMessage formatting.

## Proxy.yaml
set using `kubectl apply -f proxy.yaml -n bridgesplash`
```yaml
metadata:
  name: public
spec:
  clusterRef:
    name: bridgesplash
  replicas: 1
  service:
    type: LoadBalancer
    externalTrafficPolicy: Local
  template:
    spec:
      version:
        channel: Velocity
        name: latest
      config:
        motd: "<red>▓▒░             ⚡</red>   <gradient:red:blue><bold>BridgeSplash</bold></gradient>   <red>⚡             ░▒▓</red>
```

## velocity.toml
Inside Velocity pod filesystem.
```toml
config-version = "2.6"
bind = "0.0.0.0:25577"
# What should be the MOTD? This gets displayed when the player adds your server to
# their server list. Only MiniMessage format is accepted.
motd = "\\<red>▓▒░             ⚡\\</red>   \\<gradient:red:blue>\\<bold>BridgeSplash\\</bold>\\</gradient>   \\<red>⚡               ░▒▓\\</red>"
show-max-players = 1500
online-mode = true
force-key-authentication = true
prevent-client-proxy-connections = true
forwarding-secret-file = "/mnt/shulker/forwarding-secret/key"
player-info-forwarding-mode = "modern"
forced-hosts = {}

[servers]
        lobby = "localhost:30000"
        try = ["lobby", "limbo"]
        limbo = "localhost:30001"
[advanced]
        tcp-fast-open = true
        haproxy-protocol = false
```

### What components are involved in your issue?

Operator, Proxy Agent (Velocity)

### Version

v0.6.0

### Kubernetes Version

v1.28.3

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by jeremylvln** [comment]
If Velocity is not breaking its own config file (which I'm pretty sure it doesn't), then it means that the culprit is the init script of the Docker image I use, especially this function: https://github.com/itzg/docker-bungeecord/blob/master/scripts/run-bungeecord.sh#L180

Still not sure why, I continue...

**Comment by jeremylvln** [comment]
I thought it was you that added the

> # What should be the MOTD? This gets displayed when the player adds your server to
> # their server list. Only MiniMessage format is accepted.

Part in your `velocity.toml` snippet, but I have the same. So it's Velocity that is nuking its own configuration file. Not sure why.

**Comment by jeremylvln** [comment]
I found the issue. Shulker was creating Velocity configuration files with a version being `2.5`. This was triggering [this Velocity migration](https://github.com/PaperMC/Velocity/blob/dev/3.0.0/proxy/src/main/java/com/velocitypowered/proxy/config/migration/MotdMigration.java) that was rewriting the MOTD to be a valid MiniMessage (which it was already, that's weird).

I pushed a commit to set the version to `2.6` which should avoid running the migration so fixes the issue. I'll make a release once I confirmed the fix is working
